packages/react-composites/src/composites/MeetingComposite/ChatButtonWithUnreadMessagesBadge.tsx

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ChatMessage } from '@azure/communication-chat';
+import { IStackStyles, Stack } from '@fluentui/react';
+import { Chat20Regular } from '@fluentui/react-icons';
+import { ControlBarButtonProps } from '@internal/react-components';
+import React, { useState } from 'react';
+import { useEffect } from 'react';
+import { ChatAdapter } from '../ChatComposite';
+import { ChatButton } from './ChatButton';
+import { NotificationIcon } from './NotificationIcon';
+
+/**
+ * @private
+ */
+export interface ChatButtonLogicWrapperProps extends ControlBarButtonProps {
```

Comment:
Rename type to `ChatButtonWithUnreadMessagesBadgeProps`.

Reply:
resolved in this PR https://github.com/Azure/communication-ui-library/pull/1386 (oops I merged the wrong thing...)